coldfront/core/project/templates/project/project_request/savio/project_request_list.html

```diff
@@ -5,25 +5,25 @@
 
 
 {% block title %}
-Savio Project {{ request_filter|title }} Requests
+{{ PRIMARY_CLUSTER_NAME }} Project {{ request_filter|title }} Requests
```

Comment:
The issue was just about the title being wrong, so this is the only line that needs to be changed.

Can you undo the rest?